src/main/java/org/jitsi/jigasi/xmpp/CallControlComponent.java

```diff
@@ -277,84 +215,19 @@ protected IQ handleIQSetImpl(IQ iq)
     public IQ handleIQ(IQ iq)
         throws Exception
     {
-        try
+        org.jivesoftware.smack.packet.IQ resultIQ = null;
+        if (callControl == null)
         {
-            org.jivesoftware.smack.packet.IQ smackIq = IQUtils.convert(iq);
-
-            String fromBareJid = iq.getFrom().toBareJID();
-            if (allowedJid != null && !allowedJid.equals(fromBareJid))
-            {
-                org.jivesoftware.smack.packet.IQ error
-                    = org.jivesoftware.smack.packet.IQ
-                        .createErrorResponse(
-                                smackIq,
-                                new XMPPError(XMPPError.Condition.not_allowed));
-
-                return IQUtils.convert(error);
-            }
-            else if (allowedJid == null)
-            {
-                logger.warn("Requests are not secured by JID filter!");
-            }
-
-            if (smackIq instanceof RayoIqProvider.DialIq)
-            {
-                RayoIqProvider.DialIq dialIq = (RayoIqProvider.DialIq) smackIq;
-
-                String from = dialIq.getSource();
-                String to = dialIq.getDestination();
-
-                String roomName = dialIq.getHeader(ROOM_NAME_HEADER);
-                String roomPassword = dialIq.getHeader(ROOM_PASSWORD_HEADER);
-                if (roomName == null)
-                    throw new RuntimeException("No JvbRoomName header found");
-
-                logger.info(
-                    "Got dial request " + from + " -> " + to
-                    + " room: " + roomName);
-
-                String callResource
-                    = initNewCall(roomName, roomPassword, from, to);
-
-                callResource = "xmpp:" + callResource;
-
-                RayoIqProvider.RefIq ref
-                    = RayoIqProvider.RefIq.createResult(smackIq, callResource);
-
-                return IQUtils.convert(ref);
-            }
-            else if (smackIq instanceof RayoIqProvider.HangUp)
-            {
-                RayoIqProvider.HangUp hangUp
-                    = (RayoIqProvider.HangUp) smackIq;
-
-                String callUri = hangUp.getTo();
-                String callResource = callUri;
-
-                GatewaySession session = gateway.getSession(callResource);
-
-                if (session == null)
-                    throw new RuntimeException(
-                        "No gateway for call: " + callResource);
-
-                //hangupMap.put(gateway, smackIq.getFrom());
-
-                session.hangUp();
-
-                org.jivesoftware.smack.packet.IQ result
-                    = org.jivesoftware.smack.packet.IQ.createResultIQ(smackIq);
-
-                return IQUtils.convert(result);
-            }
-            else
-            {
-                return super.handleIQSet(iq);
-            }
+            logger.warn("Call controller not initialized");
```

Comment:
This is handled by the super.... calls, I will test it once again, but I'm sure I tested this case and was seeing the errors in the client.